File: hanzsgergo/week01/week-01-java-project/src/Functions/Factorial.java
package Functions;

//  Create the usual class wrapper and main method on your own
public class Factorial {
  public static void main(String[] args) {
    // - Create a function called `calculateFactorial()`
    //   that returns the factorial of its input
    int number = 2;
    int check = CalculateFactorial(number);

    //check
    System.out.println(check);

  }

  public static int CalculateFactorial(int num) {
    int result = 1;
    for (int i = 1; i <= num; i++) {
      result *= i;
    }
    return result;
  }
}

File: Foundation/week_02-week_04/PythonBasicsProject/week_02/functions/factorial.py
# - Create a function called `calculateFactorial()`
#   that returns the factorial of its input

num = 1


def calculate_factorial(n):
    result = 1
    for i in range(0, n+1):
        result *= i
    if n == 0:
        result = 0
    return result

print(calculate_factorial(num))

